bindings/pydeck/tests/io/test_html.py

```diff
@@ -44,10 +45,18 @@ def test_iframe_with_srcdoc():
     IPython.display.HTML = MagicMock()
     html_str = "<html></html>"
     iframe_with_srcdoc(html_str)
-    escaped_html_str = html.escape("<html></html>")
-    iframe = """<iframe src="about:blank" frameborder="0" srcdoc="{escaped_html_str}" width="100%" height=500></iframe>""".format(
-        escaped_html_str=escaped_html_str
-    )
+
+    iframe = """
+        <iframe
+            width="100%"
+            height=500
+            frameborder="0"
+            srcdoc="{escaped_html_str}"
```

Comment:
Interesting that you don't need `src`, apparently called out in the spec [here](https://html.spec.whatwg.org/#an-iframe-srcdoc-document). Good find, @ajfriend